controllers/articles.controller.js

```diff
@@ -0,0 +1,18 @@
+const { fetchArticles } = require('../models/articles.model');
+
+exports.getArticleById = (req, res, next) => {
+    let articleId;
+
+    if (isNaN(req.params.article_id)){
+        next({errorCode: 400})
```

Comment:
While this works, it wouldn't be needed take a look at the error which PSQL provide you if you input something other than a number and how you can use this for handling this error.

Reply:
I'm handling errors from PSQL, but I think in this case it is better to check if id is a number in a controller and do not create unnecessary load on the database.  But ok, ill redo this.